integration-tests/tests/src/utils/import-app.ts

```diff
@@ -24,24 +24,23 @@ export type ErrorResponse = { message: string; appId: never };
 export type ImportResponse = { appId: string; message: never };
 export type Response = ImportResponse | ErrorResponse;
 
-const getUrls = (): { appImporterUrl: string; mongodbRealmBaseUrl: string } => {
+function getUrls() {
   // Try reading the app importer URL out of the environment, it might not be accessiable via localhost
   const { appImporterUrl, mongodbRealmBaseUrl } = environment;
-  const url = typeof appImporterUrl === "string" ? appImporterUrl : "http://localhost:8091";
-
-  return { appImporterUrl: url, mongodbRealmBaseUrl: mongodbRealmBaseUrl as string };
-};
+  return {
+    appImporterUrl: typeof appImporterUrl === "string" ? appImporterUrl : "http://localhost:8091",
+    baseUrl: typeof mongodbRealmBaseUrl === "string" ? mongodbRealmBaseUrl : "http://localhost:9090",
```

Comment:
```suggestion
    mongodbRealmBaseUrl: typeof mongodbRealmBaseUrl === "string" ? mongodbRealmBaseUrl : "http://localhost:9090",
```

for consistency maybe?

Reply:
I'll change the `mongodbRealmBaseUrl` to `realmBaseUrl` to keep it consistent with the `REALM_BASE_URL` environment variable passed to the `realm-app-importer`, but I think in the context of this file, `baseUrl` can't be understood and it makes it simpler to add into the config of the `App` constructor below.

Reply:
OK 👍